src/services/auth.js

```diff
@@ -371,6 +372,45 @@ class UserService {
       user: user,
     };
   }
+
+  async changePassword(req) {
+    const { email, oldPassword, newPassword, confirmNewPassword } = req.body;
```

Comment:
The only values needed in the body are oldPassword and newPassword. We do not do password confirms on the server side. Also, the userId or identifier should be gotten from the token not the body.